packages/core/src/icon/icon.scss

```diff
@@ -2,4 +2,6 @@
   display: inline-flex;
   height: 2rem;
   width: 2rem;
+  --fill-default: currentColor;
```

Comment:
I usually put the CSS custom properties *before* the regular CSS properties. That's also the order if you sort the lines alphabetically :)